support-frontend/test/codecs/CirceDecodersTest.scala

```diff
@@ -192,7 +192,51 @@ class CirceDecodersTest extends WordSpec with MustMatchers {
           |            { "value": "25", "isDefault": true }
           |        ]
           |    }
-          |  }
+          |  },
+          |  "contributionTypes": {
+          |"GBPCountries": [
```

Comment:
I know it's a bit of a pain but it would be nice to have correct indentation here to help skim-reading the file